apps/communications/contacts/test/marionette/import_sdcard_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+
+marionette('Import from SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
```

Comment:
I don't necessary agree. I didn't want to modify the rest of the test by changing the configuraiton, so explicitly I'm asking for a specific configuration for this test.